test/e2e/installplan_e2e_test.go

```diff
@@ -60,6 +63,140 @@ var _ = Describe("Install Plan", func() {
 		TearDown(testNamespace)
 	})
 
+	When("an InstallPlan step contains a deprecated resource version", func() {
+		var (
+			pdb      policyv1beta1.PodDisruptionBudget
+			manifest string
+			counter  float64
+		)
+
+		BeforeEach(func() {
+			dc, err := discovery.NewDiscoveryClientForConfig(ctx.Ctx().RESTConfig())
+			Expect(err).ToNot(HaveOccurred())
+
+			v, err := dc.ServerVersion()
+			Expect(err).ToNot(HaveOccurred())
+
+			if minor, err := strconv.Atoi(v.Minor); err == nil && minor < 21 {
```

Comment:
this check makes sense -- are you describing that in 1.25 the API under deprecation notice would potentially be removed entirely, so these tests would not be relevant in 1.25+? Should we do `21 < minor < 25` in that case?

Reply:
This assumes we're going to revisit this in 2021 as part of the Bundle API work in order to avoid doing the work required to maintain a test webhook or to add support for arbitrary CRs to InstallPlans. If not, then this test should fail and we will be forced to deal with it then.